src/Platform/OpenGL/Shaders/lens_flare.glsl

```diff
@@ -0,0 +1,34 @@
+layout(binding = 0) uniform sampler2D lensFlareColor;
+layout(binding = 1) uniform sampler2D inputColor;
+
+uniform int ghosts; // number of ghost samples
+uniform float ghostDispersal; // dispersion factor
+uniform float uHaloWidth;
+
+in vec2 TexCoord;
+out vec4 outColor;
+
+void main()
+{
+	vec2 texcoord = TexCoord;
+	texcoord = -texcoord + vec2(1.0);
+	// ghost vector to image centre:
+	vec2 ghostVec = (vec2(0.5) - texcoord) * ghostDispersal;
+
+	// sample ghosts
+	vec4 result = vec4(0.0);
+	for (int i = 0; i < ghosts; ++i) {
+		vec2 offset = fract(texcoord + ghostVec * float(i));
+		float weight = length(vec2(0.5) - offset) / length(vec2(0.5));
+		weight = pow(1.0 - weight, 80.0);
+
+		result += texture(lensFlareColor, offset) * weight;
+	}
+
+	vec2 haloVec = normalize(ghostVec) * uHaloWidth;
+	float weight = length(vec2(0.5) - fract(texcoord + haloVec)) / length(vec2(0.5));
+	weight = pow(1.0 - weight, 75.0);
```

Comment:
pow(x, 75) is quite close to pow(x, 80) maybe make them the same and move as constant?